esp32-poe-technik.yaml

```diff
@@ -1,6 +1,16 @@
 esphome:
   name: esp32-poe-technik
   friendly_name: ESP32-PoE Technik
+  includes:
+    - src/callback_handler.h
+    - src/communication.h
+    - src/custom_climate.h
+    - src/mapper.h
+    - src/mapper.cpp
+    - src/property.h
+    - src/simple_variant.h
+    - src/type.h
+    - src/type.cpp
```

Comment:
This is not what we want. Structurally correct place is the the wp_base.yaml

Reply:
Yes, however I have the issue that I use esphome plugin, which holds all yaml files in the esphome directory. This repo I put in epshome/oneESPtoRuleThemAll. My main yaml (which holds more stuff than just the tecalor interface) is in esphome dir.
--> for not messing the whole folder structure I need to adapt the include path to /oneESPtoRuleThemAll/src/callback_handler.h etc.. 
--> Do you have a suggestion for an alternative solution?

Reply:
The thing is that the heat pump part is not meant to be used as a package by other yamls. The standard use case is to clone the repo and to build it through the esp32-poe-technik.yaml
So what I would suggest is to use symlinks in your setup that either point to the files one by one or to the src folder.

Reply:
Hm this is really unfortunate, because I was even thinking to propose to move the heatpump part into an external component [link](https://esphome.io/components/external_components). Symlinks would be very unclean. Maybe you revisit you decission?

Reply:
This can be done in the future. What I want to avoid is having a broken design due to different usage for single users

Reply:
Okay, but this makes it very unclean on my end. Would you consider the merge, if I spend the effort to make the heatpump a more generic package?

Reply:
Sorry, but I would not merge it like this. It is not the right thing to do. Just keep the changes on your side, there should not be many conflicts. You should be able to keep everything in sync easily until we have the package ready ... winter is coming anyways so we will have more time 😅